### DISCUSSION TITLE
更新umi4之后关于layout配置

### DISCUSSION THREAD

**Discussion by singgl** [umijs/umi#11519] [General]
![image](https://github.com/umijs/umi/assets/26128398/a4d6bebb-d8de-4456-b1c8-8f57096e27ae)
这是什么原因呢?

**Comment by fz6m** [comment]
文件名需要是 `app.tsx` ，同时确保导入含有 `Footer` 而不是 type ，如还有问题，需要给一个最小复现。

**Reply by singgl** [reply]
就是官方提供的
![image](https://github.com/umijs/umi/assets/26128398/1535a23c-8afa-4b08-972c-ea45de8492c2)
示例

**Comment by singgl** [comment]
![image](https://github.com/umijs/umi/assets/26128398/cc6db055-8661-4c2b-b4f9-36d5451e43d5)
引入多语音切换组件没有显示

**Reply by fz6m** [reply]
如有问题，需要给一个最小复现。

**Comment by singgl** [comment]
![image](https://github.com/umijs/umi/assets/26128398/04c86599-7803-4eb7-b7ae-d0017bc96637)
![image](https://github.com/umijs/umi/assets/26128398/be377045-78d3-4ae5-b51d-92889a883c58)

**Reply by fz6m** [reply]
给一个最小复现 repo 看看。
